SETUP.md

```diff
@@ -61,6 +61,21 @@ Assuming the repo is cloned as `Recommenders` in the local system, to install **
 
 You can specify the environment name as well with the flag `-n`.
 
+<details>
+<summary><strong><em>Java version for PySpark</em></strong></summary>
+
+**Note.** By default we use pyspark v2.4.3. It doesn't work on Java versions >8.
+
+To install Java 8 on MacOS using [asdf](https://github.com/halcyon/asdf-java):
+
+    brew install asdf
+    asdf plugin add Java
+    asdf install java adoptopenjdk-8.0.265+1
+    asdf global java adoptopenjdk-8.0.265+1
+    . ~/.asdf/plugins/java/set-java-home.zsh
```

Comment:
I haven't used a mac in a bit, but curious if asdf is needed here vs more direct use of [adoptopenjdk](https://github.com/AdoptOpenJDK/homebrew-openjdk)?
```
brew tap AdoptOpenJDK/openjdk
brew cask install adoptopenjdk8
```

Reply:
I've tried this one more time rn and when I call java -version I get this:
```
openjdk version "14.0.1" 2020-04-14
OpenJDK Runtime Environment (build 14.0.1+14)
OpenJDK 64-Bit Server VM (build 14.0.1+14, mixed mode, sharing)
```
Which stops 
So I chose asdf the first time because it set environmental variables correctly (or so I think).

I'm happy to use this solution though. What do you think should I do to fix this?

Reply:
I think typically you would run something like this (and add it in .bashrc or .bash_profile)
```export JAVA_HOME=$(/usr/libexec/java_home -v1.8);```

but if asdf handles installation without extra setup it's fine. Does this require use of z-shell?

Reply:
@loomlike can you test this out on mac and the 2.4.3 / 2.4.5 versioning issues?

Reply:
> Does this require use of z-shell?
zsh is the default shell on Catalina now I believe

Reply:
oh wow, i'm really falling behind the times in os/x world =)

Reply:
@gramhagen I tested and everything looks good. Please see my comment in the previous thread.